src/Pagination/_variables.scss

```diff
@@ -30,6 +30,9 @@ $pagination-border-color:           theme-color("gray", "border") !default;
 
 $pagination-focus-box-shadow:       $input-btn-focus-box-shadow !default;
 $pagination-focus-outline:          0 !default;
+$pagination-focus-border-width:     .125rem !default;
+$pagination-focus-color:     $primary-500 !default;
```

Comment:
nit: should this have extra indents to line up with the other variables?

Reply:
Corrected